src/Autocomplete/index.tsx

```diff
@@ -100,14 +102,25 @@ const Autocomplete: React.FC<Props> = ({
   useEffect(() => {
     let suggestions: any[] = []
     if (!result.loading && result.data) {
-      const edges = _.get(result.data, `${connectionName}.edges`)
-      suggestions = _.map(edges, (edge: any) => ({
-        label: extractLabel(edge.node),
-        value: extractValue(edge.node),
-      }))
+      if (connectionName) {
+        const edges = _.get(result.data, `${connectionName}.edges`)
+        suggestions = _.map(edges, (edge: any) => ({
+          label: extractLabel(edge.node),
+          value: extractValue(edge.node),
+        }))
+      }
+      else if (resultPath) {
+        const list = _.get(result.data, `${resultPath}`)
+        console.log(list)
+        suggestions = _.map(list, (item: any) => ({
+          label: extractLabel(item),
+          value: extractValue(item),
+        }))
+        console.log(suggestions)
+      }
```

Comment:
I think its a good idea to add an `else` that just logs a warning indicating that both `resultPath` and `connectionName` are not set